src/Bridges/Nette/Bridge.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * This file is part of the Tracy (http://tracy.nette.org)
+ * Copyright (c) 2004 David Grudl (http://davidgrudl.com)
+ */
+
+namespace Tracy\Bridges\Nette;
+
+use Nette,
+	Tracy,
+	Tracy\Helpers,
+	Tracy\BlueScreen,
+	Latte;
+
+
+/**
+ * Initializes Tracy
+ */
+class Bridge
+{
+
+	public static function initialize()
+	{
+		$blueScreen = Tracy\Debugger::getBlueScreen();
+		if (class_exists('Nette\Framework')) {
+			$bar = Tracy\Debugger::getBar();
+			$bar->info[] = $blueScreen->info[] = 'Nette Framework ' . Nette\Framework::VERSION . ' (' . Nette\Framework::REVISION . ')';
+		}
+
+		$blueScreen->addPanel(function($e) {
+			if ($e instanceof Latte\CompileException) {
+				return array(
+					'tab' => 'Template',
+					'panel' => '<p>'
+						. (is_file($e->sourceName)
+							? '<b>File:</b> ' . Helpers::editorLink($e->sourceName, $e->sourceLine)
+							: htmlspecialchars($e->sourceName, ENT_IGNORE, 'UTF-8'))
+						. '</p>'
+						. ($e->sourceCode
+							? '<pre>' . BlueScreen::highlightLine(htmlspecialchars($e->sourceCode, ENT_IGNORE, 'UTF-8'), $e->sourceLine) . '</pre>'
+							: ''),
+				);
+			} elseif ($e instanceof Nette\Neon\Exception && preg_match('#line (\d+)#', $e->getMessage(), $m)) {
+				if ($item = Helpers::findTrace($e->getTrace(), 'Nette\DI\Config\Adapters\NeonAdapter::load')) {
+					return array(
+						'tab' => 'NEON',
+						'panel' => '<p><b>File:</b> ' . Helpers::editorLink($item['args'][0], $m[1]) . '</p>'
+							. BlueScreen::highlightFile($item['args'][0], $m[1])
+					);
+				} elseif ($item = Helpers::findTrace($e->getTrace(), 'Nette\Neon\Decoder::decode')) {
+					return array(
+						'tab' => 'NEON',
+						'panel' => BlueScreen::highlightPhp($item['args'][0], $m[1])
+					);
+				}
+			}
+		});
```

Comment:
> There are things that I do not know where to put

Could you write a list of those things to https://github.com/nette/bootstrap/issues/4? Maybe someone will have a good idea.